embassy-stm32/src/ospi/mod.rs

```diff
@@ -461,6 +410,10 @@ impl<'d, T: Instance, Dma> Ospi<'d, T, Dma> {
         d7.set_as_af_pull(d7.af_num(), AFType::OutputPushPull, Pull::None);
         d7.set_speed(crate::gpio::Speed::VeryHigh);
 
+        T::REGS.cr().modify(|w| {
+            w.set_dmm(false);
```

Comment:
register writes here won't work because this is before `T::enable_and_reset();`. easiest fix is passing it as a param to `new_inner`, i'd say

Reply:
Makes sense